projects/fusion-ui/components/dropdown-option/entities/dropdown-option.ts

```diff
@@ -17,7 +17,7 @@ export interface DropdownOption {
     isSelected?: boolean;
     childOptions?: DropdownOption[];
     permissions?: string[]; // if user can see option by his permission
-    subText?: {text: string; icon?: string};
+    subText?: {text: string; icon?: string} | string;
```

Comment:
icon has the type: IconData = {iconName: string; iconVersion: string} | string. please replace icon the be this type to avoid errors when consumer put icons from: 'v1' or 'v2' folders.